xgi/generators/uniform.py

```diff
@@ -346,6 +369,47 @@ def _index_to_edge(index, n, m):
     return [(index // (n**r) % n) for r in range(m - 1, -1, -1)]
 
 
+def _index_to_edge_comb(index, n, m):
+    """Generate a hyperedge given an index in the list of possible edges.
+
+    In this function, we prohibit multiedges, so each index corresponds to a
+    unique edge.
+
+    Parameters
+    ----------
+    index : int > 0
+        The index of the hyperedge in the list of all possible hyperedges.
+    n : int > 0
+        The number of nodes
+    m : int > 0
+        The hyperedge size.
+
+    Returns
+    -------
+    list
+        The reconstructed hyperedge
```

Comment:
Would there be a better word than "reconstructed"? I'm not sure exactly what is meant